packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -36,6 +37,30 @@ However, if an error occurs during sign-in, Okta may need to display an error pa
 
 The [Brands API](/docs/reference/api/brands/) allows you to set icons, images, and colors across your Okta-hosted Sign-In Widget, error pages, email templates, and Okta End-User Dashboard all at once. You don't need to set a customized Okta URL domain. See [Customize your Okta experience with the Brands API](/docs/guides/customize-themes) for more information on this feature and how to use it.
 
+## Content Security Policy (CSP) for your custom domain
+
+If you have a [custom domain](/docs/guides/custom-url-domain/main/), you need to customize the default [CSP](https://content-security-policy.com/) for your org. Okta has a default CSP that can break customizations to Okta-hosted error pages.
```

Comment:
I'm not sure this is accurate. I think only if the org has customized sign in page they might need to customize the CSP settings and only if they add custom sources. However this might be too much detail here so I'd probably be ok leaving this as is.

Reply:
Okay. thank you.